### DISCUSSION TITLE
upgrade redoc version

### DISCUSSION THREAD

**Discussion by cfal** [fastapi/fastapi#8616] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
none
```

### Description

some users have complained of bugs with the redoc version used in fastapi, eg. the left pane cannot be expanded to show endpoints (https://github.com/Redocly/redoc/issues/1887)

the redoc URL hardcoded at openapi/docs.py (https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js) is 3 months old (2.0.0-rc.58 at the time of this writing) and it's hard to change it without adding our own redoc route.

is it possible to allow easier customization of this url, or better yet, to point it to `redoc@latest`? happy to create the PR if this seems like a good direction.

thank you!

### Operating System

Linux

### Operating System Details

Google Chrome (99.0.4844.51)

### FastAPI Version

HEAD

### Python Version

3.9.10

**Comment by Kludex** [comment, marked as answer]
You can check this page: https://fastapi.tiangolo.com/advanced/extending-openapi/#include-the-custom-docs, and use the `get_redoc_html` function. 🙏
